_data/changelogs/component-combo-box.yml

```
@@ -2,6 +2,14 @@ title: Combo box
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Removed verbose aria instructions.
```

To stay closer to plain language, I vote we replace the word "verbose" with something simpler.

```suggestion
    summary: Removed custom screen reader instructions.
```